### ISSUE TITLE
[iOS] Build fails in CI Build with `DroidKaigiMPP` xcframework

### ISSUE THREAD

**Issue by ry-itto** [DroidKaigi/conference-app-2021#502]
## Kind (Required)

- Bug report (please use the other template)
- Proposal / Discussion

 ## Overview (Required)

- In iOS App, when CI builds `Debug` or `Release` project and it depends on `DroidKaigiMPP` target, it fails.
  - Like this action https://github.com/DroidKaigi/conference-app-2021/actions/runs/932648605 .
  ```
  ...
  /Users/runner/work/conference-app-2021/conference-app-2021/ios/Sources/Model/Author.swift:1:8: error: no such module 
  'DroidKaigiMPP'
  import DroidKaigiMPP
  ...
  ```
- I tried various things, but they are not working.
  - Try to depend `AppFeature` library with `Embed & Signed`.
  - Export `AppFeature` library with `dynamic library`
  - Add `DroidKaigiMPP` xcframework to Xcode Project's dependency framework.

- This problem doesn't happen in my local environment.

I would like to investigate this problem in this issue 🙏🏼 

 ## Links	

-

**Comment by takahirom** [comment]
I will look this 👀

**Comment by takahirom** [comment]
@watanavex @myihsan let me mention for solving the issue 🙇

**Comment by takahirom** [comment]
@ry-itto Can you give me the branch or commit number that is having the problem?

**Comment by ry-itto** [comment]
This problem may be an iOS platform specific issue 🙇🏼 
I'll investigate this problem on this weekend.

If I can't solve it, then please investigate KMM module 🙏🏼 

---

@takahirom 
PR: https://github.com/DroidKaigi/conference-app-2021/pull/496
Branch:  `ry-itto:ios-repository-impl`

Above PR and branch is having this problem !

**Comment by myihsan** [comment]
It looks like a bug of `xcodebuild` (reproduce it only once by Xcode), I "fixed" it by building `DroidKaigiMPP` target first.
https://github.com/myihsan/conference-app-2021/pull/1/files
If this workaround is fine for you guys, please tell me where should I rebase to (#496 or #498?).

**Comment by ry-itto** [comment]
@myihsan 
Thanks! It looks good!

If your solution succeeded, please create new pull request (Fix CI changes only) 🙏🏼
